crystal/shivneri/config.yaml

```diff
@@ -0,0 +1,9 @@
+framework:
+  github: ujjwalguptaofficial/shivneri
+  version: 0.13.1
```

Comment:
could you please use `0.13` instead of `0.13.1` ?

Reply:
done